### ISSUE TITLE
Add API to query completed tasks

### ISSUE THREAD

**Issue by Maxr1998** [Doist/todoist-api-python#32]
Since the Sync API Python library was deprecated, I'd like to port one of my scripts to the new API library. However, I noticed the current REST API and this new library don't have endpoints to query completed tasks (the docs explicitly write ["Get active tasks"](https://developer.todoist.com/rest/v1/?python#tasks)).

Is support for such API and integration into the new library planned?

**Comment by curious-attempt-bunny** [comment]
+1 Based on polling the Get Active Tasks API it's not possible to determine whether a task was completed vs deleted.

**Comment by mikedgit** [comment]
Yes, this would enable workflows like automatically reporting out completed tasks to team members and track productivity in a more flexible way!  I suspect that once you have this in filters (which is something asked for a lot I'm sure), this will be added to the API, but definitely upvoting this.

**Comment by theptrk** [comment]
Anyone resorting to the Sync API for this?

**Comment by jkawamoto** [comment]
I'm using the Sync API to get completed tasks and opened a PR #90 to add it to this client.

**Comment by goncalossilva** [comment]
#90 was merged, thanks @jkawamoto!
